hs_core/models.py

```diff
@@ -134,7 +138,7 @@ def validate_hydroshare_user_id(value):
 
         # check the user exists for the provided user id
         if not User.objects.filter(pk=value).exists():
-            raise ValidationError(err_message)
+            raise UserValidationError(err_message)
```

Comment:
This change may cause issue where the `create()` function of the `Party` model is being used and there it probably is looking for `ValidationError` only.

Reply:
I thought it looked like we handle all Exceptions during metadata creation
https://github.com/hydroshare/hydroshare/blob/2.11/hs_core/views/__init__.py#L626-L644C52

Reply:
Also, UserValidationError is a subclass of ValidationError so the Party model would catch UserValidationError even if it `except ValidationError`

Reply:
You are right - since the new exception class is a subclass, it should work.